### PR TITLE
Add a note about cgroup v2 support in the 0.33 release

### PR DESCRIPTION
**Pull request by babsingh**
Related: https://github.com/eclipse-openj9/openj9-docs/pull/942

Signed-off-by: Babneet Singh <sbabneet@ca.ibm.com>

### CODE REVIEW

**Comment by babsingh** [comment]
fyi @pshipton @tajila.

File: doc/release-notes/0.33/0.33.md [hunk 1]
```diff
@@ -70,6 +70,13 @@ The option cannot be supported with value types.</td>
 <td valign="top">The Visual Studio redistributable files included with the build are updated to match.</td>
 </tr>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/14190">#14190</a></td>
+<td valign="top">Control groups v2 support added</td>
+<td valign="top">All versions</td>
+<td valign="top">The Linux kernel has two variants of <a href="https://man7.org/linux/man-pages/man7/cgroups.7.html">control groups (cgroups): v1 and v2</a>. Many Linux operating systems are gradually transitioning from cgroups v1 to v2 as their default choice. OpenJ9 has added cgroups v2 support which is identical to the support for cgroups v1.</td>
```
[the remark this answers is not in the document]
Comment (tajila) [reply doc/release-notes/0.33/0.33.md:77]: I'll let @pshipton have the final say on wording

**Review by tajila — APPROVED** [review]

File: doc/release-notes/0.33/0.33.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pshipton) [reply doc/release-notes/0.33/0.33.md:77]: Esther is tagged for a review of this wording in the User Guide. We should wait for that to be finalized and then use the same wording here.

[the remark this answers is not in the document]
Comment (pshipton) [reply doc/release-notes/0.33/0.33.md:77]: https://github.com/eclipse-openj9/openj9-docs/pull/942 was merged so accepting the same wording here.

**Pull request merged**